app/models/spree/flow_io_product_decorator.rb

```diff
@@ -28,9 +28,9 @@ def flow_included?(flow_exp)
       flow_data["#{flow_exp.key}.excluded"].to_i != 1
     end
 
-    def price_range(product_zone)
+    def price_range(product_zone, currencies = [])
```

Comment:
This is a good opportunity to add coverage to this legacy method 🙇🏼‍♀️

Reply:
I added a test for this new case!